source/sqlsrv/stmt.cpp

```diff
@@ -460,26 +472,25 @@ PHP_FUNCTION( sqlsrv_field_metadata )
     try {
 
     // get the number of fields in the resultset
-    num_cols = core::SQLNumResultCols( stmt TSRMLS_CC );
+    SQLSMALLINT num_cols = get_resultset_meta_data(stmt);
```

Comment:
This is where you get the full metadata and not just the number of fields, isn't it? The comment should be updated to reflect that.

Reply:
`if it exists`?